consensus/abci_info_connection.go

```diff
@@ -0,0 +1,76 @@
+package consensus
```

Comment:
[MINOR] I think `abci_state.go` refers better to the actual implementation included in this file

Reply:
👍

Reply:
Hmm I would rather to call it "info". Another state (like 5th) would be mega confusing. PLUS and this is the biggest argument, it's officially called "info" in Tendermint design. Mempool, Consensus and Info are all "reserved keywords" by Tendermint so we shall be accurate when referring to them and use the same names.